### PR TITLE
Possibility to configure temporal test server host

### PR DESCRIPTION
**Pull request by cv65kr**
## Why?
To have possibility to change default port for temporal test server.
Also:
Since output of RR is not visible in PHPunit sometimes it's hard to figure out where is the error. So you can switch test server to real temporal server and see in UI what's happened.
Alternatively Datadog offers https://github.com/temporalio/temporalite own server which also can be use for tests purposes.

### CODE REVIEW

**Comment by seregazhuk** [comment]
Maybe it can be moved to env variable in `phpunit.xml` file? 🤔 Instead of hardcoding or passing around variables.

**Comment by cv65kr** [comment]
That was my original idea, but I was a bit confused why it was hard coded on beginning 🤔 Anyway updated

File: testing/src/Environment.php [hunk 1]
```diff
@@ -44,7 +44,7 @@ public function start(string $rrCommand = null): void
 
         $this->output->write('Starting Temporal test server... ');
         $this->temporalServerProcess = new Process(
-            [$this->systemInfo->temporalServerExecutable, 7233, '--enable-time-skipping']
+            [$this->systemInfo->temporalServerExecutable, getenv('TEMPORAL_ADDRESS'), '--enable-time-skipping']
```
Comment (seregazhuk) [inline testing/src/Environment.php:47]: Maybe we can add a default value here? Like this:

```php
getenv('TEMPORAL_ADDRESS') ?: '127.0.0.1:7233`
```

Or pass the variable into `Environment::create()` constructor and hide this "default logic" there. The same about trait and `WorkflowTestCase`. Let's add deafult value there.

**Review by seregazhuk — APPROVED** [review]

File: testing/src/Environment.php (hunk 1 above)
Comment (cv65kr) [reply testing/src/Environment.php]: I added default value :)

**Comment by cv65kr** [comment]
@seregazhuk I think I found issue with CI, let's try with CI

**Review by seregazhuk — APPROVED** [review]
@cv65kr thank you! It passes all checks 💪 merging 🚀

**Pull request merged**